README.md

```diff
@@ -71,7 +73,7 @@ MIT (see [LICENSE](LICENSE))
 
 # Bad Merge Example
 
-An example of why it may be desirable to build the merged commit. Let's say you're raising a PR to implement a `subtract` function to a calculator that already does `add`ition:
+An example of why it may be desirable to build the merged commit. Let's say you're raising a PR to implement an `add` function to a calculator that already does `subtract`ion:
```

Comment:
I see what you did there 👏.  However I wonder if it reads better to rather say

> ...to a calculator that can already `subtract`

Reply:
Fixed